packages/website/docs/what-is-docsearch.md

```diff
@@ -1,37 +1,36 @@
 ---
 title: What is DocSearch?
+sidebar_label: Getting started
+authors:
+  - name: Clément Vannicatte
+    title: Software Engineer @ Algolia
+    url: https://github.com/shortcuts
 ---
 
 ## Why?
 
-We have created DocSearch because we are scratching our own itch. As
-developers, we spent a lot of time reading documentation, and it can be hard to
-find relevant information in large documentations. We're not blaming anyone
-here; building a good search is a challenge.
+We have created DocSearch because we are scratching our own itch. As developers, we spent a lot of time reading documentation, and it can be hard to find relevant information in large documentations. We're not blaming anyone here; building a good search is a challenge.
 
-It happens that we are a search company and we actually have a lot of experience
-building search interfaces. We wanted to use those skills to help others. That's
-why we created a way to automatically extract content from tech documentation
-and make it available to everyone from the first keystroke.
+It happens that we are a search company and we actually have a lot of experience building search interfaces. We wanted to use those skills to help others. That's why we created a way to automatically extract content from tech documentation and make it available to everyone from the first keystroke.
 
 ## Quick description
 
-We split DocSearch into a crawler and a frontend library. We run the crawler on
-our end every 24 hours to extract content from your website and push it to an Algolia
-index. You can then add the frontend library to your website to search in that index.
+We split DocSearch into a crawler and a frontend library.
+
+- Crawls are handled by the [Algolia Crawler][4] and sheculed to run once a week, you can then trigger new crawls yourself and monitor them directly from the [Crawler interface][5], which also offers a live editor where you can maintain your config.
+- The frontend library is built on top of [Algolia Autocomplete][6] and provide an immersive search experience through its modal.
 
 ## How to feature DocSearch?
 
-DocSearch is entirely free and automated. The one thing we'll need from you is
-to read [our checklist][2] and apply! After that, we'll share with you the
-JavaScript snippet needed to add DocSearch to your website. We ask that you keep
-the "Search by Algolia" link displayed.
+DocSearch is entirely free and automated. The one thing we'll need from you is to read [our checklist][2] and apply! After that, we'll share with you the snippet needed to add DocSearch to your website. We ask that you keep the "Search by Algolia" link displayed.
 
-DocSearch is [one of our ways][1] to give back to the open source community for
-everything it did for us already.
+DocSearch is [one of our ways][1] to give back to the open source community for everything it did for us already.
 
 You can now [apply to the program][3]
 
 [1]: https://opencollective.com/algolia
```

Comment:
We might want to talk about this with DevRel, it's been 2years since last donation

Reply:
I don't have much details on it but indeed!